src/app/other-user-profiles/other-user-profiles.component.html

```diff
@@ -0,0 +1,21 @@
+<div class="other-user-profiles mat-typography ">
+  <h1>Other user skill profiles</h1>
+  <div *ngIf="errorMessage" class="other-user-profiles-error">{{errorMessage}}</div>
+  <mat-card *ngFor="let user of permissionOwners" class="other-user-profiles-card">
```

Comment:
If there are no users who granted any permissions to the authenticated user we should show something like "There are no users who granted permissions to you".

Reply:
done